doc/ipython-notebooks/classification/Classification.ipynb

```diff
@@ -84,11 +84,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "shogun_feats_linear = sg.features(sg.csv_file(os.path.join(SHOGUN_DATA_DIR, 'toy/classifier_binary_2d_linear_features_train.dat')))\n",
-    "shogun_labels_linear = sg.labels(sg.csv_file(os.path.join(SHOGUN_DATA_DIR, 'toy/classifier_binary_2d_linear_labels_train.dat')))\n",
+    "shogun_feats_linear = sg.create_features(sg.create_csv(os.path.join(SHOGUN_DATA_DIR, 'toy/classifier_binary_2d_linear_features_train.dat')))\n",
```

Comment:
@karlnapf @vigsterkr would it make sense to rename `create_csv` to `load_csv`. It goes against the pattern but create implies (imo) that we are creating a CSV file?

Reply:
we could also name all of those `sg_*`The names before more were more ambiguous (i.e. also allow for casting/conversions). But not sure. `load_csv` is also good

Reply:
load_csv is good... as well as read_csv :) either is fine

Reply:
yea, `read_csv` would be like pandas which is better

Reply:
ok, i will change create_csv to read_csv.

Reply:
for consistency you should do the same with all the other file parsers, imo